Gemfile.lock

```diff
@@ -10,10 +10,11 @@ GIT
       rack (>= 1.2, < 3)
 
 GIT
-  remote: git://github.com/jnicklas/capybara.git
-  revision: 4b3093f4bd12a682b41d8e88d3958d33e70f4662
+  remote: https://github.com/jnicklas/capybara.git
+  revision: 3f5c36f1a45e0f05365539524ee17adb20a7c8d5
   specs:
```

Comment:
If you change master Gemfile, capybara gem line from `github: ...` to `git: ...`, bundler will refetch the repo, hence this change.